spanner/spansql/keywords.go

```diff
@@ -135,6 +135,7 @@ var funcs = map[string]bool{
 	"MAX":       true,
 	"MIN":       true,
 	"SUM":       true,
+	"AVG":       true,
```

Comment:
keep these in alphabetical order.

Reply:
I didn't notice the order rule, thanks.
https://github.com/googleapis/google-cloud-go/pull/3286/commits/64453991b15679ec55ac385d6a943f3bf999b2b2